homeassistant/components/blink/alarm_control_panel.py

```diff
@@ -40,6 +40,16 @@ def __init__(self, data, name, sync):
         self._attr_unique_id = sync.serial
         self._attr_name = f"{DOMAIN} {name}"
 
+    @property
+    def device_info(self):
```

Comment:
Please use a `DeviceInfo`  TypedDict and not a standard dictionary.
Also, please use `self._attr_device_info` rather than a property.